subsystems/intakesubsystem.py

```diff
@@ -0,0 +1,76 @@
+from commands2 import SubsystemBase
+from wpilib import PneumaticHub, PneumaticsModuleType, Solenoid, SmartDashboard
+import constants
+from util.helpfulIO import Falcon
+
+
+class IntakeSubsystem(SubsystemBase):
+    def __init__(self) -> None:
+        SubsystemBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.intakeDeployed = False  # default to intake retracted and not reversed
+        self.intakeReversed = False
+
+        self.pneumaticsHub = PneumaticHub(constants.kPneumaticsHubCanID)
```

Comment:
unused hub? why does it even need to be initialized if nothing is directly being done with it

Reply:
It is being used, the solenoids just default to it so it doesn't have to be called. Pretty sure it's required to be there though (and also for the real robot)

Reply:
if it defaults, it doesnt have to be constructed, I read the docs again (https://robotpy.readthedocs.io/projects/wpilib/en/stable/wpilib/Solenoid.html?highlight=Solenoid) and it doesnt seem to be needed....

Reply:
Maybe you're right, it doesn't explicitly say that. I had assumed that it would need to know there is a pneumatics hub to assign solenoids on it. I guess give it a shot without the constructor and see what happens